docs/README.md

```diff
@@ -1,143 +1,581 @@
 
-| Name      | Type          | Default  | Description |
-| :------------- |:-------------| :-----|:-----|
-| fallbackToWindowLocation     | `boolean`   | `false`  |If `true`, sets the href of `window.location` with the future path
-| fetchPage     | `boolean`   | `true`  | If `false`, won't fetch navigation assets in `pages-graphql`
-| page     | `string`  | --  | Name of the page that will be redirected to. Maps to a `blocks.json` block. Example: `'store.product'`
-| to     | `string`    |  --  | Alternatively to `page`, you can pass the whole URL directly instead of the page name (Useful for the `search-result`). Example: `/shirt/p?skuId=1`
-| params | `object`      |   `{}`  | Map of _parameters_ names in the path for the page and the values that should replace them. Example: `{slug: 'shirt'}`
-| query | `string`  | `''`   | String representation of the query params that will be appended to the path. Example: `skuId=231`.
-| scrollOptions | `RenderScrollOptions` | -- | After the navigation, if the page should be scrolled to a specific position, or should stay still (use `false`)
-| replace | `boolean` | `undefined` | If it should call the replace function to navigate or not
-| preventRemount | `boolean` | `false` | If `true`, only the URL will change, but not the components :exclamation: **Use with caution!**
+**Example value:**
+`"store.home"`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { page } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `production`
+
+A _boolean_ value representing whether the app is in a production workspace or not.
+
+**Example value:**
+`false`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { production } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `query`
+
+An _object_ that stores the query string values in a key-value format.
+
+**Example value:**
+`{ "foo": "bar" }`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { query } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `renderMajor`
+
+A _number_ with the major version of Render Runtime.
+
+**Example value:**
+`8`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { renderMajor } = useRuntime()
+
+  return <div>Hello</div>
+}
+
+export default MyComponent
+```
+
+### `rootPath`
+
+A _string_ with the root path of the store. It can be `undefined` if none is set.
+
+**Example value:**
+`/ar`
+
+**Usage:**
+```tsx
+import React from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+function MyComponent() {
+  const { rootPath } = useRuntime()
+
+  return <div>Hello</div>
+}
```

Comment:
Could not think of a good example here, but we should also use the `rootPath` variable in some way here.